src/core/js/models/configModel.js

```diff
@@ -16,13 +16,36 @@ export default class ConfigModel extends LockingModel {
     };
   }
 
+  /**
+   * Parse the incoming search queries for a 'lang' parameter value to return
+   */
+  getLanguageFromURL() {
```

Comment:
I like [your idea of making this more generic](https://github.com/adaptlearning/adapt-contrib-languagePicker/pull/46#discussion_r596862681) - it'd just be a question of where best to put that code - this doesn't feel like quite the right place? @oliverfoster any suggestions?

Reply:
We might also consider adding a [polyfill for URLSearchParams](https://github.com/ungap/url-search-params)